lib/components/primitive-components/Group/Group.ts

```diff
@@ -300,6 +300,10 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
       }
 
       if (job.has_error) {
+        db.pcb_autorouting_error.insert({
+          pcb_error_id: autorouting_job.autorouting_job_id,
+          message: JSON.stringify(job.error),
```

Comment:
i gave you the logic for this in the comment, `job.error?.message ?? JSON.stringify(job.error)`